app/views/catalog/_home.html.erb

```diff
@@ -1,11 +1 @@
-<div class='jumbotron'>
-  <div class='row'>
-    <div class='col-md-8 offset-md-2'>
-      <h1 class='text-center'><%= t('arclight.views.home.heading') %></h1>
-    </div>
-  </div>
-  <div class='row'>
-    <div class='col-md-6 offset-md-3'>
-        <%= render_search_bar %>
-    </div>
-</div>
+
```

Comment:
Not saying that we should do anything w/ this PR, but just wondering if we have/should create a ticket about adding content to the home page, just so it doesn't get lost.

Even if we don't get to adding the real content there, we might want to add some instructional information similar to what Blacklight does out of the box.  Not sure what folks thoughts are on that.

I can bring it up during standup, and don't think it should hold up this PR.